namco/io.md

```diff
@@ -40,6 +40,22 @@ External serial interface. Connects to motherboard serial, and exposes JVS and s
   - (internal) 1x 7pin (to power supply 12V and 5V, pinout left-right: GND GND NC GND 12V 5V)
   - (internal) 1x 4pin (to JVS expansion card, pinout left-right: V+ D+ D- V-)
 
+## H055 CARD-IF
+
+Banapass card reader. Connects to the motherboard via a custom serial port connector.
```

Comment:
Oh, does this hook up to the SERIAL3 (CARD IF)?

Reply:
It does!